### ISSUE TITLE
Introduce a requirements description template

### ISSUE THREAD

**Issue by krucod3** [eclipse-ankaios/ankaios#151]
## Description

Currently we don't have a defined requirements description template which lead to some malformed requirements using "is" or "will" instead of "shall". As proposed by @lingnoi in #150, we can introduce a template for the requirements.
@christoph-hamm proposed to use EARS (https://aaltodoc.aalto.fi/server/api/core/bitstreams/d518c3cc-4d7d-4c69-b7db-25d2da9e847f/content)

## Goals

Define a requirements template and document it in our development guidelines.

### Summary

A proposal of the template is [in the comment below](https://github.com/eclipse-ankaios/ankaios/issues/151#issuecomment-2034343406).

### Tasks

- [x] Propose the template
- [x] Discuss the proposal with the team. See [the comment below](https://github.com/eclipse-ankaios/ankaios/issues/151#issuecomment-2034343406).
- [x] Describe the template in the documentation. #230

**Comment by maturar** [comment]
I have discussed the proposal with the team and here is the proposal:
```makrdown
<Requirement title>
`swdd~<component>-<descriptive requirement id>~<version>`

Status: approved

[When <condition separated by and>] <object> shall <do something, be in state, execute a list of actions in order/in parallel…> 

Comment: 
<comment body>

Rationale: 
<rationale body>

Tags:
- <tag1>
- <tag2>
- …

Needs:
- [impl/utest/stest]

NOTE:
* Only one object is allowed.
* Format is markdown.
* Tags are the objects and subject of the requirement, which are already specified in the structural view.


Inspired by:
https://aaltodoc.aalto.fi/server/api/core/bitstreams/d518c3cc-4d7d-4c69-b7db-25d2da9e847f/content
```

**Comment by maturar** [comment]
The PR with the implementation #230 has been reviewed and merged into `main`.
